cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2017,6 +2017,15 @@ public static final class MetadataConsumer {
   public static final class SourceControlManagement {
     public static final String GIT_COMMAND_TIMEOUT_SECONDS = "source.control.git.command.timeout.seconds";
     public static final String GIT_REPOSITORIES_CLONE_DIRECTORY_PATH =
-      "source.control.git.repositories.clone.directory.path";
+            "source.control.git.repositories.clone.directory.path";
+  }
+
+  /**
+   * Constants for SQL dialect conversion service SPI
+   */
+  public static final class SqlDialectConversion {
+    public static final String EXTENSION_DIR = "sqldialect.extensions.dir";
+    public static final String CALCITE_IMPL_NAME = "calcite";
```

Comment:
I think this it's used only in extension, don't need it in commons

Reply:
Sure, removed.